hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/allocation/AllocationFileQueueParser.java

```diff
@@ -211,6 +211,9 @@ private void loadQueue(String parentName, Element element,
         isReservable = true;
         builder.reservableQueues(queueName);
         builder.configuredQueues(FSQueueType.PARENT, queueName);
+        LOG.warn("During fs2cs conversion, we cannot convert the leaf queue "
+            + queueName + " to weight mode without losing the " +
+            "reservation functionality.");
```

Comment:
Can you please change this log to appear only when the weight mode is set as the target queue mode? Using --percentage or -pc flag when launching fs2cs will result in a percentage config, where this log is misleading.